Formula/nvc.rb

```diff
@@ -0,0 +1,57 @@
+class Nvc < Formula
+  desc "VHDL compiler and simulator"
+  homepage "https://github.com/nickg/nvc"
+
+  stable do
+    url "https://github.com/nickg/nvc/releases/download/r1.0.0/nvc-1.0.0.tar.gz"
+    sha256 "a60478636268a7d1cad2d1c90e775cfd938199dba6b9c407badbef32753dc30f"
+
+    # fixes nickg/nvc#265
```

Comment:
Can you please add an explanation of why the patch is needed in a bit more depth. Thanks!